site/source/_posts/2019-11-18-nim-vs-crystal-part-1-performance-interoperability.md

```diff
@@ -0,0 +1,457 @@
+title: Nim vs Crystal - Part 1 - Performance & Interoperability
+summary: "Crystal and Nim go head-to-head to figure out the best modern, low-level programming language!  In part 1, Performance & Interoperability are reviewed."
+author: robin_percy
+categories:
+  - tutorials
+layout: blog-post
+image: '/assets/images/nim-crystal-header_blank.jpg'
+alias: news/2019/11/18/nim-vs-crystal-performance-interoperability/
```

Comment:
The alias is only needed if this content was originally published under a different URL and we want to preserve the lifetime of any link that points to that different URL.